### DISCUSSION TITLE
cara mengubah tipe data Mon Apr 06 22:19:49 PDT 2009

### DISCUSSION THREAD

**Discussion by BNCC20066** [teamalgoritma/community#553] [Q&A] [answered]
Dear All,
saya ingin mengubah tipe data, dan tidak menemukan format yang sesuai, kira-kira ada solusi untuk persoalan ini.
Terima kasih
<img width="239" alt="Screen Shot 2023-02-20 at 22 29 36" src="https://user-images.githubusercontent.com/72481877/220150168-b300cd6f-75a8-4726-a1ae-bca91898701d.png">

**Comment by jafarhf1** [comment, marked as answer]
Dear Bu Genoveva @BNCC20066 ,

Untuk merubah tipe data dengan format yang Ibu lampirkan bisa menggunakan fungsi `as.POSIXct()` dan memasukkan format yang sesuai dengan data yang Ibu miliki. Pada kasus ini, Ibu bisa menuliskannya dengan misalnya:
``` 
x <- 'Mon Apr 06 22:19:45 PDT 2009'
as.POSIXct(x, "%a %b %d %H:%M:%S PDT %Y", tz='US/Pacific')
```

Demikian yang dapat kami sampaikan, Semoga membantu.
